### PR TITLE
Implement --target-attribute-path

### PR DESCRIPTION
**Pull request by ErinvanderVeen**
This PR adds the `--target-attribute-path` to the cli, allowing the extraction of a predetermined attribute path.

### CODE REVIEW

File: nixtract/cli.py [hunk 1]
```diff
@@ -245,6 +245,11 @@ def try_queue_get():
     default="github:nixos/nixpkgs/master",
     help="The reference of the target Nix Flake",
 )
+@click.option(
+    "--target-attribute-path",
+    default="",
+    help="The the attribute path of the provided attribute set to evaluate",
```
Comment (GuillaumeDesforges) [inline nixtract/cli.py:251]:
```suggestion
    help="The attribute path of the provided attribute set to evaluate",
```

File: README.md [hunk 2]
```diff
@@ -36,13 +36,17 @@ Usage: nixtract [OPTIONS] OUTFILE
...
 Options:
-  --target-flake-ref TEXT  The reference of the target Nix Flake
-  --target-system TEXT     The system in which to evaluate the derivations
-  --n-workers INTEGER      Count of workers to spawn to describe the stream of
-                           found derivations
-  --offline                Pass --offline to Nix commands
-  -v, --verbose            Increase verbosity
-  --help                   Show this message and exit.
+  --target-flake-ref TEXT       The reference of the target Nix Flake
+  --target-attribute-path TEXT  The the attribute path of the provided
```
Comment (GuillaumeDesforges) [inline README.md:40]:
```suggestion
  --target-attribute-path TEXT  The attribute path of the provided
```

File: nixtract/find-attribute-paths.nix [hunk 3]
```diff
@@ -16,11 +17,16 @@ let
 
   # Arguments have to be taken from environment when using `nix` command
   targetFlakeRef = builtins.getEnv "TARGET_FLAKE_REF";
+  targetAttribute = builtins.getEnv "TARGET_ATTRIBUTE_PATH";
```
Comment (GuillaumeDesforges) [inline nixtract/find-attribute-paths.nix:20]: _beware my nitpicking_ :smile: 

```suggestion
  targetAttributePath = builtins.getEnv "TARGET_ATTRIBUTE_PATH";
```

File: nixtract/find-attribute-paths.nix [hunk 4]
```diff
@@ -16,11 +17,16 @@ let
...
   targetSystem = builtins.getEnv "TARGET_SYSTEM";
 
   # Get pkgs
   targetFlake = builtins.getFlake targetFlakeRef;
   targetFlakePkgs = lib.getFlakePkgs targetFlake targetSystem;
+  targetPackage =
```
Comment (GuillaumeDesforges) [inline nixtract/find-attribute-paths.nix:26]: I'm not convinced we could call it a "package".
I would rather see it as the "root" value from which we recurse.

```suggestion
  targetRootValue =
```

**Review by GuillaumeDesforges — CHANGES_REQUESTED** [review]
Looks good! Just some nitpicking on naming.

**Comment by ErinvanderVeen** [comment]
Nitpicking is good!

**Review by GuillaumeDesforges — APPROVED** [review]

**Pull request merged**